radix-engine-constants/src/lib.rs

```diff
@@ -1,18 +1,41 @@
-/// The default cost units loaned from the system to bootstrap execution (lock fee).
-pub const DEFAULT_SYSTEM_LOAN: u32 = 1_000_000;
+//==========================
+// Transaction construction
+//==========================
+
+/// The default cost unit limit.
+pub const DEFAULT_COST_UNIT_LIMIT: u32 = 100_000_000;
+
+//==========================
+// Transaction validation
+//==========================
 
-/// The default max cost unit limit for a transaction, used by transaction validator.
+pub const TRANSACTION_VERSION_V1: u8 = 1;
+pub const MAX_NUMBER_OF_INTENT_SIGNATURES: usize = 16;
+
+/// The minimum value of cost unit limit
+pub const DEFAULT_MIN_COST_UNIT_LIMIT: u32 = 1_000_000;
+
+/// The maximum value of cost unit limit
 pub const DEFAULT_MAX_COST_UNIT_LIMIT: u32 = 100_000_000;
 
-/// The default cost unit limit for a transaction.
-pub const DEFAULT_COST_UNIT_LIMIT: u32 = 100_000_000;
+/// The minimum value of tip percentage
+pub const DEFAULT_MIN_TIP_PERCENTAGE: u8 = 0;
 
-/// The default cost unit price.
-pub const DEFAULT_COST_UNIT_PRICE: &'static str = "0.0000001";
+/// The maximum value of tip percentage
+pub const DEFAULT_MAX_TIP_PERCENTAGE: u8 = u8::MAX;
```

Comment:
Max tip percentage shouldn't be 100?

Reply:
Haha, we deliberately allowed values greater than 100%.